### DISCUSSION TITLE
Combobox - Add option to allow input of text that isn't in the autocomplete list

### DISCUSSION THREAD

**Discussion by AFCMS** [tailwindlabs/headlessui#1277] [Ideas]
It would be very useful in some cases where you want to create the element if nonexistent.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your suggestion!
Much appreciated! 🙏

You can already do that today. Here is an example: https://codesandbox.io/s/stupefied-paper-ce9r5b?file=/src/App.js

**Reply by AFCMS** [reply]
Glad to know its already possible! Thank you very much!
